reference/6/Microsoft.PowerShell.Core/New-PSSessionConfigurationFile.md

```diff
@@ -68,46 +68,63 @@ Customized sessions that include the cmdlets, functions and scripts that technic
 
 ### Example 2: Restricting Language in a Session
 
+
+The first pair of commands uses the **New-PSSessionConfigurationFile** cmdlet to create two session configuration files. The first command creates a no-language file. The second command creates a restricted-language file. Other than the value of the *LanguageMode* parameter, the session configuration files are equivalent. 
+
+```powershell
```

Comment:
We cannot break these examples up like this. This is a limitation of  PlatyPS. See the style guide I referenced in the previous comment.

Reply:
Got it! Thank you so much!